core/src/main/kotlin/org/lflang/generator/cpp/CppGenerator.kt

```diff
@@ -96,6 +96,7 @@ class CppGenerator(
             )
             if (targetConfig.get(DockerProperty.INSTANCE).enabled) {
                 copySrcGenBaseDirIntoDockerDir()
+                FileUtil.copyDirectoryContents(context.fileConfig.srcPkgPath.resolve("src"), context.fileConfig.srcGenPath.resolve("src"), true)
```

Comment:
This line looks a bit sus. Why are we copying `src` into `src-gen/src`?

Reply:
`context.fileConfig.srcGenPath` points to the directory that we put the Dockerfile in. `src` is copied for the same reason as we copy `src-gen`. In the C++ target's LF files may refer to files in the `src` directory, and hence it needs to be available in the docker image.